src/commands/whois.ts

```diff
@@ -0,0 +1,60 @@
+import moment from 'moment';
+import { CommandDefinition } from '../lib/command';
+import { CommandCategory } from '../constants';
+import { makeEmbed } from '../lib/embed';
+
+const beautifiedStatus = {
+    online: "Online",
+    idle: "Idle",
+    dnd: "Do Not Disturb",
+    offline: "Offline"
+}
+
+export const whois: CommandDefinition = {
+    name: 'whois',
+    description: 'Explain the absence of release dates or ETAs',
```

Comment:
**issue:** This is the wrong description :p

Reply:
Could you provide me a description for it?

Reply:
Changed in commit d81b1f5